build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+python3 -m venv venv3
+inv deps
+source venv3/bin/activate
+pip3 install -r requirements.txt
```

Comment:
Shouldn't some of this stuff belong to setup.sh? What's inv?

Reply:
http://docs.pyinvoke.org/en/stable/index.html

Reply:
I can move the creation of venv outside but the activate needs to be done on per shell basis.